content/documentation/tutorials/nats-minio.md

```diff
@@ -0,0 +1,62 @@
++++
+date = "2015-09-27"
+title = "Using NATS with Minio"
+description = ""
+category = "tutorials"
+[menu.main]
+  name = "Using NATS with Minio"
+  weight = 4
+  identifier = "tutorial-nats-minio"
+  parent = "Tutorials"
++++
+
+# Using NATS with Minio
+
+## NATS and Minio
+[Minio](www.minio.io) is a Golang-based object store, increasingly popular for cloud native deployments. Minio is S3 compatible, and runs across a wide variety of platforms.
+
+This demonstration will show you how to run a Minio object store on a local machine, configure a local NATS instance, and finally replicate objects to other clouds.
+
+## Step 1: Install and run NATS Server
+`go get github.com/nats-io/gnatsd`
+
+## Step 2: Install Minio
+`go get github.com/minio/minio`
+
+## Step 3: Configure Minio to use NATS for event subscription
+edit `~/.minio/config.json`
+
+set `"nats"."1"."enable": true`
+
+`...
```

Comment:
This did not display correctly for me - I believe you want `text` or maybe try `json` for syntax highlighting.